src/test/java/com/wikia/webdriver/elements/common/Navigate.java

```diff
@@ -44,4 +49,20 @@ public Navigate toUrl(String url) {
     driver.get(url);
     return this;
   }
+
+  private String getQueryParams(String pageName) {
+    return pageName.equals("") || pageName.equals("/") ? "" : "?cb=" + DateTime.now().getMillis();
+  }
+
+  private String getQueryParams(String pageName, String[] queryParams) {
+    String query = this.getQueryParams(pageName);
+
+    if (!query.equals("")) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1132)